lerna.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "node_modules/lerna/schemas/lerna-schema.json",
-  "version": "0.5.3",
+  "version": "0.5.4-alpha.0",
```

Comment:
remove the alpha?

Reply:
This will get auto updated by lerna when I release